user/tests/integration/communication/startup-slos/slos.cpp

```diff
@@ -0,0 +1,11 @@
+#include "application.h"
```

Comment:
Let's also move these out of `communication` folder into its own e.g. `slos` to make things neater, as `communication` within Device OS repo specifically means anything dealing with cloud commnucations and cloud primitives: pubsub, functions, variables etc.